helpers/authentication.js

```diff
@@ -10,18 +9,7 @@ const isJWT = (req) => {
 };
 
 const isAuthenticated = (req) => {
-    if(!isJWT(req)) {
-        return Promise.resolve(false);
-    }
-
-    return api(req).post('/authentication', {json: {
```

Comment:
This allowed re-authentification in case a cookie was missing, right? Why was this removed

Reply:
I misunderstood the code and reverted these changes.